### PR TITLE
Improve Speedometer layout

### LINKED ISSUE #58: Improve speedometer
Improve the speedometer so that it is more apealing for the medical worker. It is gonna be improved by trying to implement https://github.com/anastr/SpeedView/blob/d3c568da943d9871bfd1f4c87a5a7665a4a068dd/README.md .

### PR DESCRIPTION
**Pull request by JihanP**
### Changes

1. Update design of the speedometer.
2. Adjusted UI to make room for speedometer.

### Issues
Resolves #58

### CODE REVIEW

File: android/app/src/main/java/com/example/medcarapp/ManualControl.java [hunk 1]
```diff
@@ -50,13 +52,13 @@ protected void onCreate(Bundle savedInstanceState) {
             public void onMove(int angle, int strength) {
                 int adjustedAngle = adjustAngle(angle);
                 int adjustedSpeed = adjustSpeed(strength, angle);
+                int speed = speedometerSpeed(strength,angle);
```
Comment (Marble879) [inline android/app/src/main/java/com/example/medcarapp/ManualControl.java]: This variable name could be changed to speedometerSpeed for improved readability.

**Review by Marble879 — APPROVED** [review]
Everything runs and looks great! I just have one minor suggestion which could improve the readability of the code. Great job :+1:

**Review by puerto-hub — APPROVED** [review]
The speedometer looks great!

**Review by NilsDunlop — APPROVED** [review]
This is easily going into my top 10 best code compilation! Well done guys 📈

File: android/app/src/main/java/com/example/medcarapp/ManualControl.java (hunk 1 above)
Comment (MarkusJ299) [reply android/app/src/main/java/com/example/medcarapp/ManualControl.java]: That's true, changed it.

**Pull request merged**